sizzlingstats/SizzlingStats.cpp

```diff
@@ -131,6 +131,15 @@ void SizzlingStats::PlayerHealed( int entindex, int amount )
 	data.m_pExtraData->healsrecv += amount;
 }
 
+void SizzlingStats::PlayerArrowed( int patientEntIndex, int medicEntIndex, float distance )
+{
+	playerAndExtra_t medic_data = m_PlayerDataManager.GetPlayerData(medicEntIndex);
+	medic_data.m_pExtraData->arrowslanded += 1;
+
+	playerAndExtra_t data = m_PlayerDataManager.GetPlayerData(medicEntIndex);

```

Comment:
typo here, I think you meant patientEntIndex.

And from the comment about what happens if a player fires an arrow and leaves, maybe add a  m_PlayerDataManager.IsValidPlayer(ent_index) check here. Nothing bad will happen without the check, but add it for sanity when reading this code.